src/games/strategy/engine/framework/startup/ui/PBEMSetupPanel.java

```diff
@@ -204,6 +205,7 @@ private void loadForumPosters(final GameData data) {
     forumPosters.add((IForumPoster) findCachedOrCreateNew(NullForumPoster.class));
     forumPosters.add((IForumPoster) findCachedOrCreateNew(AxisAndAlliesForumPoster.class));
     forumPosters.add((IForumPoster) findCachedOrCreateNew(TripleAWarClubForumPoster.class));
```

Comment:
should we remove the war club post now, or in a different PR? 
NullForumPoster probably should go to.